app/config/question-bank.js

```diff
@@ -616,17 +614,54 @@ const questionBank = {
           answers: [
             {
               key: 'smaller-abattoir-A1',
-              value: 'Yes'
+              value: 'Yes',
+              yarKeysReset: ['fruitStorage']
             },
             {
               key: 'smaller-abattoir-A2',
               value: 'No',
-              redirectUrl: 'project-items',
+              redirectUrl: 'fruit-storage',
               yarKeysReset: ['otherFarmers']
             }
           ],
           yarKey: 'smallerAbattoir'
         },
+        {
+          key: 'fruit-storage',
+          order: 70,
+          title: 'Do you want to build new controlled atmosphere storage for top fruit?',
+          hint: {
+            text: 'Fruit that grows on trees, for example apples, pears, quinces, medlars, plums, peaches, apricots and cherries'
+          },
+          url: 'fruit-storage',
+          baseUrl: 'fruit-storage',
+          backUrl: 'smaller-abattoir',
+          nextUrl: 'solar-PV-system',
+          preValidationKeys: [],
+          type: 'single-answer',
+          minAnswerCount: 1,
+          classes: 'govuk-radios--inline govuk-fieldset__legend--l',
+          validate: [
+            {
+              type: 'NOT_EMPTY',
+              error: 'Select yes if you want to build new controlled atmosphere storage for top fruit'
+            }
+          ],
+          answers: [
+            {
+              key: 'fruit-storage-A1',
+              value: 'Yes',
+              yarKeysReset: ['projectItems']
```

Comment:
Is there a need to reset projectItems here instead of on the smallerAbattoir page?

Also, is there a need to reset at all?

Reply:
user does not see projectitems question when they click yes so I though it'd be good to clean answer in case they go through 'No' option and switch the other journey.